### PR TITLE
Feature/us1422 navigate to corkboard

### PR DESCRIPTION
**Pull request by danrye**

### CODE REVIEW

File: Gateway/CorkboardAPI/CorkboardAPI.Tests/Controllers/ValuesControllerTest.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+        [Test]
+        public void Post()
+        {
+            // Arrange
+            ValuesController controller = new ValuesController();
+
+            // Act
+            controller.Post("value");
+
+            // Assert
```
Comment (andycanterbury) [inline Gateway/CorkboardAPI/CorkboardAPI.Tests/Controllers/ValuesControllerTest.cs]: This test has no Asserts. Also the Controller method being tested is empty. Is this unfinished?

File: Gateway/CorkboardAPI/CorkboardAPI.Tests/Controllers/ValuesControllerTest.cs [hunk 2]
```diff
@@ -0,0 +1,76 @@
...
+        [Test]
+        public void Put()
+        {
+            // Arrange
+            ValuesController controller = new ValuesController();
+
+            // Act
+            controller.Put(5, "value");
+
+            // Assert
```
Comment (andycanterbury) [inline Gateway/CorkboardAPI/CorkboardAPI.Tests/Controllers/ValuesControllerTest.cs]: This test has no Asserts. Also the Controller method being tested is empty. Is this unfinished?

File: Gateway/CorkboardAPI/CorkboardAPI.Tests/Controllers/ValuesControllerTest.cs [hunk 3]
```diff
@@ -0,0 +1,76 @@
...
+        [Test]
+        public void Delete()
+        {
+            // Arrange
+            ValuesController controller = new ValuesController();
+
+            // Act
+            controller.Delete(5);
+
+            // Assert
```
Comment (andycanterbury) [inline Gateway/CorkboardAPI/CorkboardAPI.Tests/Controllers/ValuesControllerTest.cs]: This test has no Asserts. Also the Controller method being tested is empty. Is this unfinished?

**Comment by andycanterbury** [comment]
Is the /Areas/Help section needed? Looks like an auto generated thing and it exposes a lot of info that probably doesn't need to be public.

File: Gateway/CorkboardAPI/CorkboardAPI/Controllers/HomeController.cs [hunk 4]
```diff
@@ -0,0 +1,18 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+using System.Web.Mvc;
+
+namespace MongoAPITest.Controllers
+{
+    public class HomeController : Controller
```
Comment (andycanterbury) [inline Gateway/CorkboardAPI/CorkboardAPI/Controllers/HomeController.cs]: Is this controller used or just more auto generated stuff?

File: Gateway/CorkboardAPI/CorkboardAPI/Controllers/PostsController.cs [hunk 5]
```diff
@@ -0,0 +1,165 @@
...
+                    }
+                }
+
+
+                return newList;
+            }
+
+        }
+
+        public IHttpActionResult GetPost(int id)
```
Comment (andycanterbury) [inline Gateway/CorkboardAPI/CorkboardAPI/Controllers/PostsController.cs]: No longer used?

File: Gateway/CorkboardAPI/CorkboardAPI/Controllers/ValuesController.cs [hunk 6]
```diff
@@ -0,0 +1,40 @@
...
+
+        // GET api/values/5
+        public string Get(int id)
+        {
+            return "value";
+        }
+
+        // POST api/values
+        public void Post([FromBody]string value)
+        {
```
Comment (andycanterbury) [inline Gateway/CorkboardAPI/CorkboardAPI/Controllers/ValuesController.cs]: Empty method. Is this used?

File: Gateway/CorkboardAPI/CorkboardAPI/Controllers/ValuesController.cs [hunk 7]
```diff
@@ -0,0 +1,40 @@
...
+        }
+
+        // PUT api/values/5
+        public void Put(int id, [FromBody]string value)
+        {
```
Comment (andycanterbury) [inline Gateway/CorkboardAPI/CorkboardAPI/Controllers/ValuesController.cs]: Empty method. Is this used?

File: Gateway/CorkboardAPI/CorkboardAPI/Controllers/ValuesController.cs [hunk 8]
```diff
@@ -0,0 +1,40 @@
...
+        }
+
+        // DELETE api/values/5
+        public void Delete(int id)
+        {
```
Comment (andycanterbury) [inline Gateway/CorkboardAPI/CorkboardAPI/Controllers/ValuesController.cs]: Empty method. Is this used?

File: Gateway/CorkboardAPI/CorkboardAPI/Models/CorkboardNeedPost.cs [hunk 9]
```diff
@@ -0,0 +1,12 @@
+﻿using System;
+using System.Collections.Generic;
+using System.Linq;
+using System.Web;
+
+namespace MongoAPITest.Models
+{
+    public class CorkboardNeedPost : CorkboardPost
+    {
```
Comment (andycanterbury) [inline Gateway/CorkboardAPI/CorkboardAPI/Models/CorkboardNeedPost.cs]: Empty class?

File: Gateway/CorkboardAPI/CorkboardAPI/Views/Home/Index.cshtml [hunk 10]
```diff
@@ -0,0 +1,24 @@
+﻿<div class="jumbotron">
```
Comment (andycanterbury) [inline Gateway/CorkboardAPI/CorkboardAPI/Views/Home/Index.cshtml]: If this is an API project, do we need any of the stuff in /Views?

File: website/app/corkboard/corkboard.controller.js [hunk 11]
```diff
@@ -0,0 +1,61 @@
...
+
+      vm.opened = true;
+    }
+
+    function getCorkboardPosts($scope, $http)
+    {
+      vm.corkboardListings = CorkboardListings.query();
+    }
+
+    /*vm.trimCorkboardText = function(string)
```
Comment (andycanterbury) [inline website/app/corkboard/corkboard.controller.js]: Remove commented code

File: website/app/corkboard/corkboard.service.js [hunk 12]
```diff
@@ -0,0 +1,13 @@
+'use strict()';
+(function(){
+
+    angular.module('crossroads.corkboard').factory('CorkboardListings', CorkboardListings);
+
+    CorkboardListings.$inject = ['$resource'];
+
+    function CorkboardListings($resource){
+        return $resource('http://localhost:65341/api/posts');
```
Comment (andycanterbury) [inline website/app/corkboard/corkboard.service.js]: hardcoded localhost. Probably won't work when deployed.

File: website/protractorWin.conf.js [hunk 13]
```diff
@@ -0,0 +1,11 @@
+exports.config = {
+    seleniumAddress: 'http://localhost:4444/wd/hub',
```
Comment (andycanterbury) [inline website/protractorWin.conf.js]: Localhost

**Comment by danrye** [comment]
This PR is for master, we'll want this against development
